<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomVariablePageTemplate</class>
 <widget class="QWidget" name="CustomVariablePageTemplate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CustomVariableTemplate</string>
  </property>
  <widget class="QLabel" name="label_43">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_pName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>25</number>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_44">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_pStoreBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAlphabetically</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="m_pID">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>999999999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pResizeButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Resize Variables</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>m_pStoreBox</tabstop>
  <tabstop>m_pName</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
